parent/pom.xml

```diff
@@ -56,10 +56,29 @@
         <wss4j.osgi.export>org.apache.wss4j.*</wss4j.osgi.export>
         <wss4j.osgi.dynamic.import />
         <wss4j.manifest.location>${project.build.outputDirectory}/META-INF/MANIFEST.MF</wss4j.manifest.location>
+        <jakarta.xml.bind-api.version>3.0.1</jakarta.xml.bind-api.version>
+        <jakarta.xml.soap-api.version>2.0.1</jakarta.xml.soap-api.version>
+        <jakarta.mail.version>2.0.0</jakarta.mail.version>
+        <jaxb-runtime.version>3.0.2</jaxb-runtime.version>
     </properties>
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>com.sun.mail</groupId>
+                <artifactId>jakarta.mail</artifactId>
+                <version>${jakarta.mail.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>jakarta.xml.bind</groupId>
+                <artifactId>jakarta.xml.bind-api</artifactId>
+                <version>${jakarta.xml.bind-api.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>jakarta.xml.soap</groupId>
+                <artifactId>jakarta.xml.soap-api</artifactId>
+                <version>${jakarta.xml.soap-api.version}</version>
+            </dependency>
```

Comment:
Remove

Reply:
org.glassfish.jaxb:jaxb-runtime can not be removed from ws-security-stax/pom.xml
it is required for KerberosTest.java to run successfully.

All other changes are complete.

Reply:
Could it be changed to runtime scope instead? and finally, can you create a WSS JIRA for this change and reference the JIRA number in the commit message? Then we should be good to merge.

Reply:
Change complete.  
Created https://issues.apache.org/jira/browse/WSS-694 Move wss4j to native jakarta namespace.  Added it to the PR commit msg.